Source/Views/TableViews/SPCopyTable.m

```diff
@@ -1314,6 +1314,25 @@ - (BOOL)shouldUseFieldEditorForRow:(NSUInteger)rowIndex column:(NSUInteger)colIn
 	return NO;
 }
 
+/**
+ * Determine whether to use the sheet for editing; do so if the multipleLineEditingButton is enabled,
+ * or if the column was a blob or a text, or if it contains linebreaks.
+ */
```

Comment:
Comment here looks to not be associated with the function

Reply:
Oops! Reviewed comment....